src/bodywork/cli/cli.py

```diff
@@ -358,16 +344,12 @@ def deployment(args: Namespace) -> None:
         print_warn("Please specify --name for the deployment job.")
         sys.exit(1)
     if command == "create":
+        load_kubernetes_config()
         if not async_workflow:
-            pass_through_args = Namespace(
-                git_url=git_url,
-                git_branch=git_branch,
-                bodywork_docker_image=image,
-            )
             print_info("Using local workflow controller - retries inactive.")
-            workflow(pass_through_args)
+            run_workflow(git_url, git_branch, docker_image_override=image)
         else:
-            load_kubernetes_config()
+            print_info("Using async workflow controller.")
```

Comment:
`"Using asynchronous workflow controller."`

I don't think we need to shorten it.